lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -10,6 +10,13 @@ if Enum.all?(
   defmodule Membrane.RTC.Engine.Endpoint.HLS do
     @moduledoc """
     An Endpoint responsible for converting incoming tracks to HLS playlist.
+
+    This module require these plugins:
+    * membrane_http_adaptive_stream_plugin,
+    * membrane_mp4_plugin,
+    * membrane_aac_plugin,
+    * membrane_aac_format,
```

Comment:
This dependency is actually not necessary, it should be automatically fetched with `membrane_aac_plugin`